src/hotspot/os/posix/perfMemory_posix.cpp

```diff
@@ -71,8 +72,8 @@ static char* create_standard_memory(size_t size) {
 
   // commit memory
   if (!os::commit_memory(mapAddress, size, !ExecMem)) {
-    if (PrintMiscellaneous && Verbose) {
-      warning("Could not commit PerfData memory\n");
+    if (log_is_enabled(Debug, perf)) {
+      log_debug(perf)("Could not commit PerfData memory");
```

Comment:
You don't need the `log_is_enabled` check with a single logging statement as the `log_debug` is a macro that already contains the check.